backend/config/middleware.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
       }
     },
     cors: {
-      origin: ['http://localhost:8000'],
+      origin: ['http://localhost:8000', 'http://localhost:1337'],
```

Comment:
use "*" here to allow all origins and remove the two urls, if we do any CORS configuration we do it in the K8s ingress